_drafts/course-materials/geog-4100-5100/intro-knitr-rmd/rmarkdown-04-intro-to-code-chunks.Rmd

```diff
@@ -7,7 +7,11 @@ authors: [Leah Wasser, NEON Data Skills]
 category: [course-materials]
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
 class-lesson: ['intro-rmarkdown-knitr']
-permalink: /course-materials/intro-rmarkdown-code-chunks
+<<<<<<< HEAD:_drafts/course-materials/geog-4100-5100/intro-knitr-rmd/rmarkdown-04-intro-to-code-chunks.Rmd
```

Comment:
that's so weird. i've fixed that a few times. something is up with the way i'm resolving merge conflicts.